packages/contracts/test/plugins/governance/multisig/multisig.ts

```diff
@@ -656,6 +657,82 @@ describe('Multisig', function () {
         ).not.to.be.reverted;
       });
 
+      it('reverts if `_msgSender` is not listed in the current block although he was listed in the last block', async () => {
+        await ethers.provider.send('evm_setAutomine', [false]);
+        const expectedSnapshotBlockNumber = (
+          await ethers.provider.getBlock('latest')
+        ).number;
+
+        // Transaction 1 & 2: Add signers[1] and remove signers[0]
+        const tx1 = await multisig.addAddresses([signers[1].address]);
+        const tx2 = await multisig.removeAddresses([signers[0].address]);
+
+        // Transaction 3: Expect the proposal creation to fail for signers[0] because he transferred the tokens in transaction 1
```

Comment:
Not part of the multisig instead?

Reply:
Fixed in c6f8f3753887947f7d29e1aa70901892ac29b8f6